code/src/Components/Artists.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+// The Artist component //
+// exoprt beofre const //
+const Artists = (props) => {
+  console.log(props);
+  return (
+    <>
+      {props.artists.map((artist) => {
+        return <h3>{artist.name}</h3>
+      })}
+    </>
```

Comment:
Additionally, this was a good stack overflow question about comma separation between multiple artists: 
https://stackoverflowteams.com/c/technigo/questions/2239